_meetings/roundtable.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Software and Computing Roundtable"
+author: "Graeme Stewart"
+layout: plain
+---
+
+<p align=center>
+<img src="/images/Roundtable-banner.jpg"
+  alt="SIDIS-logo" style="display: inline-block; width:80%; margin:3%">
+</p>
+
+The *Software and Computing Roundtable* is a general series of meetings on topical
+matters for the high-energy and nuclear physics community, which is organised
+jointly by Jefferson Lab, Brookhaven National Lab and the
+HEP Software Foundation.
+
+The Roundtable has historically had particularly strong links to the nuclear
+physics community, but is now expanding to be the perfect meeting point
+with high-energy physics.
+
+* [Roundtable Home](https://www.jlab.org/software-and-computing-round-table)
+* [Roundtable Meetings in 2021](https://indico.jlab.org/event/420/)
+* [Roundtable Meetings in 2020](https://indico.jlab.org/event/356/)
+
+If you would like to suggest a topic for the Roundtable, please contact [HSF
+Coordination](mailto:hsf-coordination@googlegroups.com). The current HSF
+representative is [Graeme Stewart](mailto:graeme.andrew.stewart@cern.ch).
+
+<p align=center>
+<img src="/images/JLab_logo.png"
+  alt="SIDIS-logo" style="display: inline-block; width:25%; margin:3%">
+<img src="/images/hsf_logo_angled.png"
+  alt="HSF-logo" style="display: inline-block; width:20%; margin:3%">
+<img src="/images/BNL-logo.png"
```

Comment:
Do you have the BNL logo file somewhere already, or it is missing in this PR?

Reply:
It's already there as they have had GSoC projects in the past